### ISSUE TITLE
Why is the app location for the main process hardcoded in dev.mts?

### ISSUE THREAD

**Issue by jahan-addison** [Kiyozz/electron-esbuild#44]
See here: https://github.com/Kiyozz/electron-esbuild/blob/94c4a8d837b015e345eb9efca73dcd98e862df59/packages/electron-esbuild/src/commands/dev.mts#L26

Why is this hardcoded when you can change the `output` directory in the electron-esbuild.config.yaml? 

This makes it not possible to use any other `output` directory other than `dist/main` for the main process.

**Comment by Kiyozz** [comment]
Good catch. It should be the output from the yaml file!

**Comment by Kiyozz** [comment]
Coming in 6.0.0

output is now an object.

from
```yaml
mainConfig:
  # ...
  output: dist/main
```
to
```yaml
mainConfig:
  # ...
  output:
    dir: dist/main
    filename: main.js
```

**Comment by Kiyozz** [comment]
Published in 6.0.0
